server/protocols/webadmin/webadmin-mailbox/src/main/java/org/apache/james/webadmin/routes/DomainQuotaRoutes.java

```diff
@@ -247,6 +261,13 @@ public void defineGetQuotaCount() {
     }
 
     private String checkDomainExist(Request request) {
+        if (!isVirtualHostingSupported()) {
+            throw ErrorResponder.builder()
+                .statusCode(HttpStatus.METHOD_NOT_ALLOWED_405)
+                .type(ErrorType.WRONG_STATE)
+                .message("Domain Quota configuration not supported")
```

Comment:
maybe you can explain: "... when virtual hosting is desactivated". Or Even "Please use global quota configuration instead".

Reply:
Or both!